fv3core/fv3core/testing/translate_dyncore.py

```diff
@@ -125,6 +125,7 @@ def __init__(
         # TODO: Fix edge_interpolate4 in d2a2c_vect to match closer and the
         # variables here should as well.
         self.max_error = 2e-6
+        self.ignore_near_zero_errors["wsd"] = 1e-18
```

Comment:
Why aren't we using overrides yaml here?

Reply:
My understanding is yaml overrides are only for platform-dependent (e.g. gpu vs cpu) issues, whereas this is a more fundamental description of the errors for this case.